integration/template/azure_operator_charts_values.go

```diff
@@ -24,6 +24,7 @@ var AzureOperatorChartValues = `Installation:
           CIDR: "10.0.0.0/16"
           ResourceGroup: "godsmack"
           VirtualNetwork: "godsmack"
+          VirtualNetworkGateway: "GatewayTest"
```

Comment:
TODO: align with godsmack vpn gateway

Reply:
@r7vme do you know how will the host cluster VPN Gateway will be named ?

Reply:
https://github.com/giantswarm/giantnetes-terraform/pull/111/files#diff-22469f75baefa6503494b40c9b8b860bR11

godsmack-vpn-gateway